### ISSUE TITLE
Scrollbar in main body

### ISSUE THREAD

**Issue by Mhirji** [mrin9/RapiDoc#129]
Hi, 

We are looking at moving to Rapidoc and were able to get it up and running very easily, which was a good thing :)  I have however noticed that in as we are using it as part of a larger page, in what gets rendered, the main-content css class has set the overflow-y to scroll.  This creates an in body scrollbar which I would like to remove/hide, but cannot seem to override it.  Have tried to via my own css but no luck.  Any ideas?  

On a side note, I also see that the browser scrollbar is being styled via the body and html, the following is injected into the page:

<style>@font-face{font-family:'rapidoc';src:url(rapidoc-regular.woff2) format("woff2");font-weight:400;font-style:normal}@font-face{font-family:'rapidoc';src:url(rapidoc-semi-bold.woff2) format("woff2");font-weight:500;font-style:normal}@font-face{font-family:'roboto-mono';src:url(roboto-mono-regular.woff2) format("woff2");font-weight:400;font-style:normal}@font-face{font-family:'roboto-mono';src:url(roboto-mono-bold.woff2) format("woff2");font-weight:500;font-style:normal}body,html{width:100%;height:100%;margin:0;padding:0}body::-webkit-scrollbar,html::-webkit-scrollbar{width:8px;height:8px}body::-webkit-scrollbar-track,html::-webkit-scrollbar-track{background-color:transparent}body::-webkit-scrollbar-thumb,html::-webkit-scrollbar-thumb{background-color:rgba(0,0,0,0.4);border-radius:3px}
</style>

Would it be possible to opt into the scrollbar styling?  Not sure if this could be part of one of the options or not.

Thanks!!

**Comment by mrin9** [comment]
RapiDoc is a [custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements), And custom elements are style encapsulated  so you wont be able to inject any of your custom style.

However, I will add some scrollbar styling for webkit based browsers that will match the selected theme.

**Comment by mrin9** [comment]
have added scrollbar styling for webkit based browsers in this commit e7018da8bcf3880cf80254c6591d5324aa6bfc86

you can test it either by building rapidoc from source or take it from the `.dist` folder

**Comment by Mhirji** [comment]
Thanks!!  Will test it today.

**Comment by Mhirji** [comment]
@mrin9 , I tested the change and now the in-page scrollbar has gone which is great.  Regarding the <style> tab which gets injected into the <head> tag that is still there.  I did find a workaround though . . . I added a new style tag to my app which gets placed in the body, so after Rapidoc's and overrides the body and html styling the library puts in.  Now the main browser scrollbar renders as I want it so that works too!  Thanks for putting in this patch.
